### ISSUE TITLE
`example-get-started`: bar plot for feature importance

### ISSUE THREAD

**Issue by dberenbaum** [iterative/example-repos-dev#135]
Bar plots were added in https://github.com/iterative/dvc-render/issues/8. Should we switch the feature importance plot from image to bar plot? I'm not sure it's worth it since then we will have no static image plots.

**Comment by shcheklein** [comment]
Yes, may be can come with some other plots that can be reasonable for this workflow instead of removing the feature importance. Anything that comes to your mind @dberenbaum @daavoo ?

At the end it would be nice to have more plots I think.

**Comment by daavoo** [comment]
We could plot the distribution of samples across target labels (0, 1) and/or splits (train, test). 
Those are usually represented as bar plots and would be associated with a different stage (`prepare`?)

**Comment by dberenbaum** [comment]
I would prefer to convert feature importance to a bar plot since we have support for it, and then add another image plot.

One idea is a [SHAP summary plot](https://christophm.github.io/interpretable-ml-book/shap.html#shap-summary-plot), which is a more robust feature importance method:

![image](https://user-images.githubusercontent.com/2308172/190666280-c0035772-3fdd-48bf-ad53-a6e9fa9eea7e.png)

It doesn't hurt to also keep the traditional feature importance as a bar plot since all of these methods have pros and cons, and it's can help to look at more than one method.

**Comment by shcheklein** [comment]
Okay, sounds good, we can try both. I like @daavoo 's suggestion since it's way simpler. I would add another image too though, I think it's good to have more images.

Let's take this when we are done with the global/flexible plots iteration and https://github.com/iterative/example-repos-dev/pull/117 is merged?

**Comment by shcheklein** [comment]
We can prioritize this in docs planning that @jorgeorpinel is preparing as a task that one of the people from the bigger "docs" group can take (including me, I would be happy to do this).

**Comment by dberenbaum** [comment]
I started on the SHAP one in https://github.com/iterative/example-repos-dev/pull/136, so anyone can feel free to pick up from there. There's a SHAP package, so it's not difficult to add.

Having some sample distribution plot is a good idea, although I have a couple concerns about the suggested bar plot:
1. It won't change between experiments.
2. Since the data is binary, a bar plot is likely not as interesting or realistic as the others.
I don't think these are blockers, but maybe if someone works on these, they can try to find ways to make it more interesting.

A histogram of predictions from training and test data might be another good bar plot.
